lib/xlsx/xform/comment/style/vml-position-xform.js

```diff
@@ -0,0 +1,39 @@
+const BaseXform = require('../../base-xform');
+
+class VmlPositionXform extends BaseXform {
+  constructor(model) {
+    super();
+    this._model = model;
+  }
+
+  get tag() {
+    return this._model && this._model.tag;
+  }
+
+  render(xmlStream, model, type) {
+    if (model === type[2]) {
+      xmlStream.leafNode(this.tag);
+    } else if (this.tag === 'x:SizeWithCells' && model === type[1]) {
```

Comment:
this if is unnecessary, both variants are the same.

Reply:
@Siemienik This is necessary to distinguish the three situations of location attribute.